packages/images-plugin/lib/finder.js

```diff
@@ -359,15 +424,19 @@ export default class ImageElementFinder {
     // screenshot size like `@driver.window_rect #=>x=0, y=0, width=1080, height=1794` and
     // `"deviceScreenSize"=>"1080x1920"`
     if (screenWidth !== shotWidth && screenHeight !== shotHeight) {
-      log.info(`Scaling screenshot from ${shotWidth}x${shotHeight} to match ` +
-               `screen at ${screenWidth}x${screenHeight}`);
+      log.info(
+        `Scaling screenshot from ${shotWidth}x${shotHeight} to match ` +
+          `screen at ${screenWidth}x${screenHeight}`
+      );
       imgObj = imgObj.resize(screenWidth, screenHeight);
 
       scale.xScale *= (1.0 * screenWidth) / shotWidth;
       scale.yScale *= (1.0 * screenHeight) / shotHeight;
     }
 
-    b64Screenshot = (await imgObj.getBuffer(imageUtil.MIME_PNG)).toString('base64');
+    b64Screenshot = (await imgObj.getBuffer(imageUtil.MIME_PNG)).toString(
```

Comment:
it seems the max allowed line length in prettier is too low

Reply:
It should be 80 chars.  but it also does extra wrapping.

Reply:
in the time of text terminals this probably made sense, but for modern displays I prefer to increase this value to 120 chars